Apps/UnitTests/X11/App.cpp

```diff
@@ -5,11 +5,12 @@
 #undef None
 #include "../Shared/Tests.cpp"
 
-static const char* s_applicationName = "Babylon Native Unit Tests";
-static const int width = 640;
-static const int height = 480;
-static const char* wmDeleteWindowName = "WM_DELETE_WINDOW";
-
+namespace {
+    constexpr char* s_applicationName = "Babylon Native Unit Tests";
+    constexpr int width = 640;
+    constexpr int height = 480;
+    constexpr char* wmDeleteWindowName = "WM_DELETE_WINDOW";
```

Comment:
constexpr doesn't replace const (common misunderstanding). You still need the const.

Reply:
There are two constness modifiers for raw char pointer. One is for the constness of the char itself and one for the constness of the pointer. Before you didn't have a constness modifier for the pointer itself and thus you can assign the pointer to something else. Without constexpre, you can fix this by doing something like this:
```
const char* const foo = "asdf";
```

With constexpr, you don't need to put the const modifier after the * to disambiguate.

```
constexpr const char* foo = "asdf";
```

Reply:
Also, you should remove the `s_` Hungarian notation or put them on all of the variables.

```suggestion
    constexpr const char* applicationName = "Babylon Native Unit Tests";
    constexpr int width = 640;
    constexpr int height = 480;
    constexpr const char* wmDeleteWindowName = "WM_DELETE_WINDOW";
```